checkstyle/checkstyle-suppressions.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+    "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+    "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+    <suppress files="[a-zA-Z0-9]*IT.java"         checks="AbbreviationAsWordInName"/>
```

Comment:
Nit, should it be prefixed with  `[/\\]test[/\\]src[/\\]java[/\\]` (also for the other IT, Test checks)?